package.json

```diff
@@ -3,9 +3,32 @@
   "version": "1.0.0",
   "description": "Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.",
   "main": "src/index.js",
+  "type": "module",
   "scripts": {
-    "start": "node ./src/index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "NODE_ENV=production node --exec babel-node ./src/index.js",
```

Comment:
please remember to put cross-env before NODE_ENV

Reply:
copy that!